examples/gallery/histograms/scatter_and_histograms.py

```diff
@@ -0,0 +1,64 @@
+"""
+Scatter plot with histograms
+----------------------------
+To create a scatter plot with histograms at the sides of the plot one
+can use :meth:`pygmt.Figure.plot` in combination with
+:meth:`pygmt.Figure.histogram`.
+"""
+
+import numpy as np
+import pygmt
+
+np.random.seed(19680801)
+
+# generate random data
```

Comment:
So far I know, we have no convention on starting comments with an upper or a lower case letter. But having this consistent within one example would be nice. So feel free to change it the other way around 🙂.

```suggestion
# Generate random data
```